src/ledger/test/LedgerTxnTests.cpp

```diff
@@ -2659,20 +2667,27 @@ TEST_CASE("LedgerTxnRoot prefetch", "[ledgertxn]")
         SECTION("prefetch more than ENTRY_CACHE_SIZE entries")
         {
             LedgerTxn ltx2(root);
-            REQUIRE(root.prefetch(keysToPrefetch) == keysToPrefetch.size());
+            REQUIRE(root.prefetchClassic(keysToPrefetch) ==
+                    keysToPrefetch.size());
             ltx2.commit();
         }
     };
 
     SECTION("default")
     {
-        runTest(Config::TESTDB_DEFAULT);
+        runTest(getTestConfig(0, Config::TESTDB_ON_DISK_SQLITE));
```

Comment:
Both the "default" and "BucketListDB" section can just use `auto cfg = getTestConfig()`, no need to specialize anything.